blocks/README.md

```diff
@@ -180,27 +292,45 @@ function edit( props ) {
+Alternatively, you can create your own toolbar controls by passing an array of
+`controls` as a prop to the `BlockControls` component. Each control should be
+an object with the following properties:
 
 - `icon: string` - Slug of the Dashicon to be shown in the control's toolbar button
 - `title: string` - A human-readable localized text to be shown as the tooltip label of the control's button
 - `subscript: ?string` - Optional text to be shown adjacent the button icon as subscript (for example, heading levels)
 - `isActive: ?boolean` - Whether the control should be considered active / selected. Defaults to `false`.
 
-To create divisions between sets of controls within the same `BlockControls` element, passing `controls` instead as a nested array (array of arrays of objects). A divider will be shown between each set of controls.
+To create divisions between sets of controls within the same `BlockControls`
+element, passing `controls` instead as a nested array (array of arrays of
+objects). A divider will be shown between each set of controls.
 
 ### `Editable`
 
-Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users the option to add emphasis to content or links to content. It behaves similar to a [controlled component](https://facebook.github.io/react/docs/forms.html#controlled-components), except that `onChange` is triggered less frequently than would be expected from a traditional `input` field, usually when the user exits the field.
+Render a rich
+[`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content),
+providing users the option to add emphasis to content or links to content. It
+behaves similarly to a
+[controlled component](https://facebook.github.io/react/docs/forms.html#controlled-components),
+except that `onChange` is triggered less frequently than would be expected from
+a traditional `input` field, usually when the user exits the field.
 
 The following props are made available:
 
-- `inline: boolean` - If true, only inline elements are allowed to be used in inserted into the text, effectively disabling the behavior of the "Enter" key.
-- `placeholder: string` - A text hint to be shown to the user when the field value is empty, similar to the [`input` and `textarea` attribute of the same name](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/HTML5_updates#The_placeholder_attribute)
-- `value: string` - Markup value of the editable field. Only valid markup is allowed, as determined by `inline` value and available controls.
-- `onChange: Function` - Callback handler when the value of the field changes, passing the new value as its only argument.
+- `inline: boolean` - If true, only inline elements are allowed to be used in
```

Comment:
I think it's `multiple` prop (which takes a tagName) now as it's inline by default.